atlasdb-feign/src/main/java/com/palantir/atlasdb/http/AtlasDbFeignTargetFactory.java

```diff
@@ -37,7 +37,10 @@
 import feign.jaxrs.JAXRSContract;
 
 public final class AtlasDbFeignTargetFactory {
-    private static final Request.Options DEFAULT_FEIGN_OPTIONS = new Request.Options();
+    // add some padding to the feign timeout, as in many cases lock requests default to a 60 second timeout,
+    // and we don't want it to exactly align with the feign timeout
+    private static final Request.Options DEFAULT_FEIGN_OPTIONS = new Request.Options(
+            10_000, 65_000);
```

Comment:
I guess we don't have a reference to the default lock request timeout (`AtlasDbConstants.DEFAULT_TRANSACTION_LOCK_ACQUIRE_TIMEOUT_MS`) so it's inconvenient to try to perform math on that.